src/goodbase/goodbase_cli.py

```diff
@@ -441,25 +438,11 @@ def main(
         output_format=output_format,
     )
 
-    display_variant_name_checks = display_variant_name
-
-    if ctx.get_parameter_source("display_variant_name") == ParameterSource.DEFAULT:
-        for (
-            project_regex,
-            default_display_variant_name_regexes,
-        ) in PROJECT_REGEX_TO_DEFAULT_DISPLAY_VARIANT_NAME_REGEXES.items():
-            if re.match(project_regex, evg_project):
-                LOGGER.debug(
-                    "Found default build variant display name regexes for evg project",
-                    evg_project_regex=project_regex,
-                    default_display_variant_name_regexes=default_display_variant_name_regexes,
-                )
-                display_variant_name_checks = default_display_variant_name_regexes
-                break
+    if build_variant is not None:
+        build_checks = BuildChecks(build_variant_regex=build_variant)
+    elif display_variant_name is not None:
+        build_checks = BuildChecks(display_variant_regex=display_variant_name)
```

Comment:
Instead of having `build_variant` silently overriding `display_variant_name` when both are present, I think we should exit and let user know that these two options are mutually exclusive.

Reply:
That's fair. That was what I first tried but I got intimidated by how many tests failed 😅

Reply:
I see. Let me know if you need any help about those tests.